CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 0.1.2
+
+- Added base action class with common code. Removed duplicate code from other actions
+- Added host_get_id action action to return the id of a Zabbix Host
+- Added host_update_status action to change the status of a Zabbix Host
+- Added maintenance_create_or_update action to create a maintenance window or update one if it already exists
+- Added maintenance_delete action to delete a maintenance window
+
```

Comment:
Make sure to give yourself credit at the bottom of the list:

"Contributed by Brad Bishop (Encore Technologies)"